pkg/controllers/disruption/consolidation.go

```diff
@@ -192,6 +193,49 @@ func (c *consolidation) computeConsolidation(ctx context.Context, candidates ...
 		results.NewNodeClaims[0].Requirements.Add(scheduling.NewRequirement(v1beta1.CapacityTypeLabelKey, v1.NodeSelectorOpIn, v1beta1.CapacityTypeSpot))
 	}
 
+	results.NewNodeClaims[0].InstanceTypeOptions = instanceTypesOptionsFilteredByPrice
+	return Command{
+		candidates:   candidates,
+		replacements: results.NewNodeClaims,
+	}, nil
+}
+
+// Allow spot consolidation if SpotConsolidation featureflag is set to true by customers:
```

Comment:
optional: Consider restructuring this into a numerical set of things that need to be true for spot-to-spot consolidation to execute like

```go
// Compute command to execute spot-to-spot consolidation if:
// 1. The SpotToSpotConsolidation feature flag is set to true
// 2. ...
```

Reply:
Addressed.